modules/create_label.rb

```diff
@@ -9,7 +9,37 @@ def create_label
 
     label = Label.new(title, color)
     @labels.push(label)
+    save_label(@labels, './data/labels.json')
     puts '=====Label created successfully====='
     puts "\n"
   end
+
+  def label_info(book)
+    if @labels.empty?
+      puts 'No labels are available. Please create a label first.'
+      create_label
+    else
+      display_labels
+      puts '\nChoose a label for this book or enter "new" to add a new label:'
```

Comment:
Please remove \n from this line. It is displaying on the frontend UI.

Reply:
done